doc/release-notes.md

```diff
@@ -1,6 +1,245 @@
 # release notes
 
 ## current release
+- release date: 2023-04-XX
+- status: available
+- overview of changes (See [Ticket 431](https://github.com/PediatricOpenTargets/ticket-tracker/issues/431) for additional details):
+  - MAJOR UPDATE
+    - This release adds the following data: 
+      - 744 methylation array data samples
+        - 728 PBTA samples (17 normal, 711 tumor)
+        - 16 TARGET samples (6 normal, 10 tumor)
+      - 1264 RNA-seq samples
+        - 1264 PBTA samples (3 normal, 1261 tumor)
+      - 2759 WGS/WXS samples
+        - 3 Kentucky samples (3 normal)
+        - 2441 PBTA WGS samples (1171 normal, 1270 tumor)
+        - 315 PBTA WXS samples (160 normal, 155 tumor)
```

Comment:
@zzgeng can you update this to current Ns - there is no Kentucky in this release, it is within PBTA, and we might have ~4 more PBTA in addition. thanks